akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -309,14 +319,37 @@ private[akka] class RemoteActorRefProvider(
       "remote-deployment-watcher")
 
   /** Can be overridden when using RemoteActorRefProvider as a superclass rather than directly */
-  protected def showDirectUseWarningIfRequired() = {
+  protected def warnIfDirectUse() = {
     if (remoteSettings.WarnAboutDirectUse) {
       log.warning(
         "Using the 'remote' ActorRefProvider directly, which is a low-level layer. " +
         "For most use cases, the 'cluster' abstraction on top of remoting is more suitable instead.")
     }
   }
 
+  // Log on `init` similar to `warnIfDirectUse`.
+  def warnIfUseUnsafeWithoutCluster(): Unit =
+    if (!settings.HasCluster) {
+      val msg =
+        if (remoteSettings.UseUnsafeRemoteFeaturesWithoutCluster)
+          "`akka.remote.use-unsafe-remote-features-without-cluster` has been enabled."
+        else
+          "Using Akka Cluster is recommended if you need remote watch and deploy."
+      log.warning(s"Cluster not in use - {}", msg)
+    }
+
+  /** Logs if deathwatch message is intentionally dropped. To disable
+   * warnings set `akka.remote.warn-unsafe-watch-without-cluster` to `off`
+   * or use Akka Cluster.
+   */
+  private[akka] def warnIfUnsafeWithoutClusterAttempted(
+      watchee: ActorRef,
+      watcher: ActorRef,
+      action: Option[String]): Unit =
+    if (warnOnUnsafeRemote) {
+      log.warning("Dropped {}: remote watch disabled for [{} -> {}]", action.getOrElse(":x"), watcher, watchee)
```

Comment:
when is action None? could we always provide something that is more useful than `:x`?

Reply:
`action` is defined when originating from `RemoteWatcher.watch/unwatch` 
and `None` when originating from `RARP.isWatchIntercepted` where we only know it is either a watch or unwatch (new trait added to cover both) so we fabricate the action in that condition.

Reply:
That `:x` is a mistaken typo, fixed.

Reply:
ok, but then you could just pass in that string instead of using an Option

Reply:
Thanks, updated.